src/components/tabController/TabBarItem.tsx

```diff
@@ -187,40 +179,63 @@ export default function TabBarItem({
     };
   });
 
+  const pressStyle = useAnimatedStyle(() => {
+    return {
+      backgroundColor: isPressed.value ? activeBackgroundColor : backgroundColor,
+      opacity: isPressed.value ? activeOpacity : 1
+    };
+  });
+
   const _style = useMemo(() => {
     const constantWidthStyle = itemWidth.current ? {flex: 0, width: itemWidth.current} : undefined;
-    return [styles.tabItem, style, constantWidthStyle];
+    return [styles.tabItem, style, constantWidthStyle, pressStyle];
   }, [style]);
 
+  const gesture = Gesture.Tap()
+    .maxDuration(60000)
+    .onEnd(() => {
+      if (!ignore) {
+        setCurrentIndex(index);
+      }
+
+      props.onPress && runOnJS(props.onPress)(index);
```

Comment:
I see this is the same behavior as before, but if a tab is ignored why do we press it?